cmds/core/tr/tr.go

```diff
@@ -45,7 +45,28 @@ import (
 	flag "github.com/spf13/pflag"
 )
 
-var delete = flag.BoolP("delete", "d", false, "delete characters in SET1, do not translate")
+var del = flag.BoolP("delete", "d", false, "delete characters in SET1, do not translate")
+
+type command struct {
+	stdin  io.Reader
+	stdout io.Writer
+	del    bool
+	tr     *transformer
+}
+
+func newCommand(args []string, del bool) (*command, error) {
```

Comment:
This function seems a bit awkward to me. Could it just be:
newCommand(in io.Reader, out io.Writer, args []string, del bool) 
it would make the test a bit less awkward looking since it would not have to change the command struct
after calling newCommand.